src/javascript/JContent/ContentNavigation/jContentHeader.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+let JContentReversedLogo = () => (
```

Comment:
Could you align the naming? Is this JContentHeader or JContentReversedLogo?
Also the file is named jContentHeader all other files seems to be named with a capital J